app/services/notification_generator.rb

```diff
@@ -7,6 +7,10 @@ def call
     sync_notifications
   end
 
+  def singular_call(child, approval)
```

Comment:
Can you tell me more about this method and what you're using it for?

Reply:
This is something I accidentally left over. I've removed it.